lib/install/management/appliance.go

```diff
@@ -631,3 +632,17 @@ func (d *Dispatcher) makeSureApplianceRuns(conf *metadata.VirtualContainerHostCo
 
 	return fmt.Errorf("could not obtain IP address information from appliance: %s", updateErr)
 }
+
+func (d *Dispatcher) createVolumeStores(conf *metadata.VirtualContainerHostConfigSpec) error {
```

Comment:
the appliance.go file is for appliance VM related operations. Would suggest move to other file, store_files, where is for the datastore file deletion right now, but can be extended for more datastore files operation.